src/mpi/datatype/looputil.c

```diff
@@ -214,8 +214,8 @@ static int external32_float_convert(char *dest_buf,
  * - Assumes that the segment has been allocated.
  *
  */
-int MPIR_Segment_init(const void *buf,
-                      MPI_Aint count, MPI_Datatype handle, struct MPIR_Segment *segp)
+static int segment_init(const void *buf,
+                        MPI_Aint count, MPI_Datatype handle, struct MPIR_Segment *segp)
 {
```

Comment:
Can we inline `segment_init ` ? `MPIR_Segment_alloc` may be called at perf-critical path, right ?

Reply:
Done.